e2e/cypress/integration/ui/settings/04_project_mgmt.spec.ts

```diff
@@ -0,0 +1,132 @@
+import forEach = require("cypress/types/lodash/forEach");
```

Comment:
❓ What would this be needed for? AFAICT the `.forEach`s below are just [standard JS](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#Syntax) 🤔

Reply:
yeah that seemed to auto-import itself for some reason 🤷‍♀